.storybook/cookie-policy.ts

```diff
@@ -0,0 +1,50 @@
+const COOKIE_ACCEPT_GDPR_NAME = 'GDPR';
+const COOKIE_ACCEPT_VALUE = 'accepted';
+
+function setGDPRCookie(expirationInDays = 365) {
+    const d = new Date();
+    d.setTime(d.getTime() + expirationInDays * 24 * 60 * 60 * 1000);
+    const expires = `expires=${d.toUTCString()}`;
+
+    document.cookie = `${COOKIE_ACCEPT_GDPR_NAME}=${COOKIE_ACCEPT_VALUE};${expires};path=/`;
+}
+
+function hasGdprCookie() {
+    return document.cookie.includes(`${COOKIE_ACCEPT_GDPR_NAME}=${COOKIE_ACCEPT_VALUE}`);
+}
+
+function showGDPRPrompt() {
+    const root = window.document.createElement('div');
+    const rootStyle = window.document.createAttribute('style');
+    rootStyle.value =
+        'position:fixed; background: #333333;width: 100%; bottom: 0; color: antiquewhite;padding: 15px;';
+
+    root.setAttributeNode(rootStyle);
+
+    window.document.body.append(root);
+    root.innerHTML = 'This site uses cookies to understand how you use the site.&nbsp;';
+
+    const acceptButton = window.document.createElement('button');
+    acceptButton.innerHTML = 'Accept Cookies';
+    acceptButton.type = 'button';
+    root.appendChild(acceptButton);
```

Comment:
this could be defined in the previous innerHTML assignment:

```js
root.innerHTML = 'This site bla bla bla. <button id="accept-cookies">Accept cookies</button>';
```

then the click listener should be assigned to the button:
```js
document.getElementbyId('accept-cookies').addEventListener('click', () => {
    setGDPRCookie();
    location.reload();
});
```